accounts/rpcserver.go

```diff
@@ -132,6 +132,14 @@ func (s *RPCServer) UpdateAccount(ctx context.Context,
 	return marshalAccount(account), nil
 }
 
+// UpdateBalance adds or deducts an amount from an existing account in the
```

Comment:
just an idea for standard PR flow: we always try to (as much as possible at least) to keep commits ordered such that things are working commit by commit. For 2 reasons: 1) reverting back to any single commit isnt a problem and 2) review story telling. 

so the ideal flow for RPC additions is:
1) add DB/Service functionality along with any tests for those (in small PR world, that could honestly be the whole PR - since that is like 1 unit of code that is easy to reason about and also lets us continue the on the accounts SQL PR while we iterate on the second unit here which is proto/CLI design).
2) add proto definitions & call the new code from the rpc server code
3) add CLI calling code

Reply:
Thanks a lot for the suggestion! I've tried to follow this with the introduction of the new #973 & #974 PRs.